src/System.IO.Pipelines/System/Threading/ThreadPoolScheduler.cs

```diff
@@ -0,0 +1,83 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+
+namespace System.Threading
+{
+    internal sealed class ThreadPoolScheduler : Scheduler
+    {
+        public override void Schedule(Action action)
+        {
+#if NETCOREAPP2_1
+            // Queue to low contention local ThreadPool queue; rather than global queue as per Task
+            Threading.ThreadPool.QueueUserWorkItem(_actionAsTask, action, preferLocal: true);
+#elif NETSTANDARD2_0
+            Threading.ThreadPool.QueueUserWorkItem(_actionAsTask, action);
+#else
+            Task.Factory.StartNew(action);
```

Comment:
Should this just be QueueUserWorkItem?

Reply:
Doesn't exist on `netstandard1.1` came in at `netstandard2.0` 😢